Makefile

```
@@ -50,7 +50,7 @@ pull_xblock_translations:  ## pull xblock translations via atlas
 
 clean_translations: ## Remove existing translations to prepare for a fresh pull
 	# Removes core edx-platform translations but keeps config files and Esperanto (eo) test translations
-	find conf/locale -mindepth 1 -maxdepth 1 -type d -a ! -name eo -exec rm -rf {} +
+	find conf/locale/ -type f \! -path '*/eo/*' \( -name '*.mo' -o -name '*.po' \) -delete
```

Thanks @timmc-edx. 

Would you mind making the command a bit more "simple":

conf/locale/en/LC_MESSAGES/.gitignore


```suggestion
	find conf/locale/ -mindepth 2 -type f ! -path '*/eo/*' ! -path '*.gitignore' -delete
```

Shell can be very tricky once we move beyond simple `ls` and `cd`.

---

It would remove the parenthetical, but it would make it more sensitive to the specifics of what kinds of non-translation files we're trying not to delete. I like the idea of deleting just the mo/po files to see if we can get away with that.

---

Makes sense. Thanks @timmc-edx. Shell is hard, hopefully this one gets us through.